src/main/java/ReviewResponse.java

```diff
@@ -2,6 +2,8 @@
 import lombok.Getter;
 import lombok.ToString;
 
```

Comment:
Violation of Single Responsibility Principle (SRP) as this class seems to be handling multiple responsibilities. (Severity: medium)